pkg/serdes.go

```diff
@@ -118,6 +118,9 @@ func imageFromBuffer(buffer []byte, transform *ARAwareTransform, aspectRatio flo
 		panic("Bit depth not set")
 	}
 
+	// Release the vips image, will free underlying buffers without having to resort to the GC
+	img.Close()
```

Comment:
GC work was a bit off here, because go didn´t know of the huge underlying buffers (I presume). We know we're done here with the VIPS object so we might as well simplify everything